ethereum_history_api/oracles/src/fixtures.ts

```diff
@@ -11,16 +11,26 @@ export async function loadFixture<T>(chain: string, hardFork: string, fixtureNam
   return fixture.result;
 }
 
-export async function loadBlockFixture(chain: string, hardFork: string, fixtureName: string): Promise<Block> {
-  return loadFixture<Block>(chain, hardFork, fixtureName, 'eth_getBlockByHash');
+export async function loadBlockFixture(
+  chain: string,
+  hardFork: string,
+  fixtureName: string,
+  includeTransactions = false
+): Promise<Block> {
+  const blockNumber = FIXTURES[chain][hardFork][fixtureName].blockNumber;
+  if (includeTransactions) {
+    return loadFixture<Block>(chain, hardFork, fixtureName, `eth_getBlockByHash_${blockNumber}_includeTransactions`);
+  }
+  return loadFixture<Block>(chain, hardFork, fixtureName, `eth_getBlockByHash_${blockNumber}`);
```

Comment:
Let's not use if's without elses for anything but error cases. Those two cases are both success, so let's use if with an else to make it more clean